parachains/runtimes/testing/rococo-parachain/build.rs

```diff
@@ -1,25 +1,26 @@
-// Copyright 2019-2021 Parity Technologies (UK) Ltd.
-// This file is part of Cumulus.
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
We're not putting the year anymore?

I think all the licenses should be updated in a dedicated PR, as this distracts from other changes here and is incomplete in itself. We have a [license scanner](https://github.com/paritytech/license-scanner) tool for this which we already used in Polkadot. cc @rzadp

Reply:
Not really sure that it distracts from the changes. Yes we are not putting the year anymore.

Reply:
Hey, feel free to ask me to set up the licence scanner CI check here if it has proven useful in Polkadot.